src/containers/Detail/index.tsx

```diff
@@ -0,0 +1,120 @@
+import { FaUser, FaTruck } from 'react-icons/fa';
+import styles from './styles.module.scss';
+import { AiTwotoneCalendar, AiFillPrinter } from 'react-icons/ai';
+import { ImLocation } from 'react-icons/im';
+import { Paper, Button, Grid, Select, Textarea, Stack } from '@mantine/core';
+import CardInfo from './components/CardInfo';
+import TableDetail from '../Detail/components/TableDetail';
+import PaymentInfo from '../Detail/components/PaymentInfo';
+export default function Detail() {
+  return (
+    <Paper shadow="sm" p="lg">
+      <h2>Order Details</h2>
+      <Paper p="lg" className={styles.cardHeader}>
+        <Grid columns={12} justify="Space-between">
+          <Grid.Col md={6} lg={6}>
+            <span>
+              <AiTwotoneCalendar className={styles.calendarIcons} />
+              <b>Thurs, April 21, 2022, 9:40PM</b>
```

Comment:
This can be replace with `Text` component from `Mantine` with `weight={600}`

Reply:
done